x-pack/plugins/observability/public/hooks/use_chart_theme.tsx

```diff
@@ -22,7 +22,7 @@ export function useChartTheme(): PartialTheme[] {
         chartMargins: {
           left: 10,
           right: 10,
-          top: 10,
+          top: 35,
```

Comment:
I needed to add to increase the top margin as the number of alerts was not hidden 
<img width="180" alt="Screenshot 2023-01-13 at 17 21 35" src="https://user-images.githubusercontent.com/6838659/212368792-a03188ae-0140-47ef-b256-5523e66887ab.png">